src/protocol/v911s_nrf24l01.c

```diff
@@ -0,0 +1,313 @@
+/*
+ This project is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ Deviation is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with Deviation.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+ 
+ #ifdef MODULAR
+  //Allows the linker to properly relocate
+  #define V911S_Cmds PROTO_Cmds
+  #pragma long_calls
+#endif
+#include "common.h"
+#include "interface.h"
+#include "mixer.h"
+#include "config/model.h"
+#include "config/tx.h" // for Transmitter
+#include "music.h"
```

Comment:
you didn't reference music functions. I am also thinking to cleanup all existing reference to Music module in protocols, which doesn't really make sense to have protocol to control the user experience.

Reply:
A few protocols need audio functions, I agree it's useless here.